tests/ptf_runner.py

```diff
@@ -34,4 +34,7 @@ def ptf_runner(host, testdir, testname, platform_dir=None, params={},
     if device_sockets:
         cmd += " ".join(map(" --device-socket {}".format, device_sockets))
 
+    if timeout:
```

Comment:
Is `0` ever a valid timeout value (like "no timeout") in ptf?

Reply:
0 is no timeout in ptf

Reply:
In that case I think the check needs to be something like this to avoid implicitly treating 0 as `false`:
```
if timeout >= 0:
```

Reply:
I just looked up the ptf code. Apparently, it throws up a warning if we try to use 0 as a parameter and then the test will proceed as no timeout. so I am going to make the check (timeout > 0)

Reply:
or just leave it the way it is

Reply:
I think that way it is is OK in that case!